src/codeql.ts

```diff
@@ -65,13 +72,46 @@ libraryPathDependencies: codeql-${language}`
 
   const databaseSHA = getDatabaseSHA(databaseName);
 
-  await exec(codeql, [
-    "query",
-    "run",
-    `--database=db`,
-    `--output=${bqrs}`,
-    queryFile,
-  ]);
+  if (query !== undefined) {
+    await exec(codeql, [
+      "query",
+      "run",
+      `--database=db`,
+      `--output=${bqrs}`,
+      queryFile,
+    ]);
+  } else if (queryPack !== undefined) {
+    await exec(codeql, [
+      "database",
+      "run-queries",
+      "--additional-packs",
+      queryPack,
```

Comment:
This is the path to the directory containing the `queryPack`? If so, this is fine.

Reply:
Yes, that's right.